src/features/views/components/EmptyView.tsx

```diff
@@ -78,8 +40,9 @@ const EmptyView: FunctionComponent<EmptyViewProps> = ({ orgId, view }) => {
               <Box marginTop={2}>
                 <PersonSelect
                   name="person"
-                  onChange={(person) => {
-                    addFirstRowMutation.mutate(person);
+                  onChange={async (person) => {
+                    await model.deleteContentQuery();
+                    await model.addPerson(person);
                   }}
```

Comment:
also the difference in how clear the wording is, how much easier it is to read the new style!